modules/web/client/src/main/scala/observe/ui/components/MainApp.scala

```diff
@@ -133,15 +133,16 @@ object MainApp extends ServerEventHandler:
   private def enforceStaffRole(ssoClient: SSOClient[IO]): IO[Option[UserVault]] =
     ssoClient.whoami
       .flatMap: userVault =>
-        if LinkingInfo.developmentMode then userVault.pure[IO]
```

Comment:
No need forr a dev mode?

Reply:
WDYM?

Reply:
The logic has changed. Observe still lets you log in while in development with non-staff users. But if the user has the STAFF role, it will still attempt to switch to it. This is because the ODB checks the user is logged in with the STAFF role to allow setting the QA state of a dataset.